### PR TITLE
Add CVE-2019-8331 for bootstrap-sass

### PR DESCRIPTION
**Pull request by reynhout**
Noticed this CVE was not in the database. Let me know if I missed anything. Thanks!

### CODE REVIEW

File: gems/bootstrap-sass/CVE-2019-8331.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+gem: bootstrap-sass
+cve: 2019-8331
+url: https://github.com/twbs/bootstrap-sass/releases/tag/v3.4.1
```
Comment (reedloden) [inline gems/bootstrap-sass/CVE-2019-8331.yml]: https://blog.getbootstrap.com/2019/02/13/bootstrap-4-3-1-and-3-4-1/ seems to give more details.

File: gems/bootstrap-sass/CVE-2019-8331.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+cvss_v2: 4.3
+cvss_v3: 6.1
+
+patched_versions:
+  - '>= 3.4.1'
+
+related:
+  url:
+    - https://blog.getbootstrap.com/2019/02/13/bootstrap-4-3-1-and-3-4-1/
```
Comment (reedloden) [inline gems/bootstrap-sass/CVE-2019-8331.yml]: Remove, as moved to top... or replace this with https://github.com/twbs/bootstrap-sass/releases/tag/v3.4.1

**Comment by reedloden** [comment]
What about https://github.com/twbs/bootstrap-rubygem/releases/tag/v4.3.1 ? Can we add a separate entry with this same CVE for https://rubygems.org/gems/bootstrap ?

Also, seems we need entries for another XSS vuln in v4.1.2 and v3.4.0, as per https://blog.getbootstrap.com/2018/07/12/bootstrap-4-1-2/ and https://blog.getbootstrap.com/2018/12/13/bootstrap-3-4-0/, respectively. Could you help with that?

File: gems/bootstrap-sass/CVE-2019-8331.yml (hunk 1 above)
Comment (reynhout) [reply gems/bootstrap-sass/CVE-2019-8331.yml]: Makes sense. I wasn't sure it was clear enough that issues with the larger Bootstrap project will apply to the `bootstrap-sass` gem, so I chose the link that showed the explicit connection first, and added the details to `related`. I'll swap them around and push changes in a moment.

Comment (reynhout) [reply gems/bootstrap-sass/CVE-2019-8331.yml]: Done

**Comment by reynhout** [comment]
Re: the other vulns mentioned: Sure, I can take a look at those today.

**Comment by reedloden** [comment]
@reynhout awesome, thanks!

**Pull request merged**